dev/Microsoft.Process.Environment/EnvironmentManager.cpp

```diff
@@ -159,6 +299,46 @@ namespace winrt::Microsoft::ProjectReunion::implementation
         throw hresult_not_implemented();
     }
 
+    std::wstring EnvironmentManager::GetPath() const
+    {
+        std::wstring path;
+        if (m_Scope == Scope::Process)
+        {
+            path = GetProcessEnvironmentVariable(L"Path");
+        }
+        else
+        {
+            wil::unique_hkey environmentVariableKey = GetRegHKeyForEVUserAndMachineScope();
+
+            DWORD sizeOfEnvironmentValue;
```

Comment:
talking about the variable sizeOfEnvironmentValue?

That is true, most people use a suffix.  I prefer prefix because it matches what we would say in English.  When talking about code we say "This is the size of the environment value."  Of course, it could go the other way around and say "This is the environment value size."

I guess I think in the former than the latter.  However, this is kinda like hungarian notation where we know it is the size of something.